src/ReactiveUI.Tests/CommandBindingTests.cs

```diff
@@ -34,7 +34,8 @@ public FakeView()
             ViewModel = new FakeViewModel();
         }
 
-        object IViewFor.ViewModel {
+        object IViewFor.ViewModel
```

Comment:
Turns out, not all the code is consistent on new lines - this came from .editorconfig changes and I think is what was the case on rebracer - can look into it if we don't like this style though.